perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -62,17 +62,41 @@ def Run(benchmark_spec):
     A list of sample.Sample objects.
   """
   vms = benchmark_spec.vms
-  if not vms[0].IsReachable(vms[1]):
-    logging.warn('%s is not reachable from %s', vms[1], vms[0])
+  results = []
+  for origin_index in [0, 1]:
```

Comment:
Ah, I guess CheckPrerequisites is outdated and only supports flags, not configs. In the long term, I think it makes sense to move the call of CheckPrerequisites after the benchmark spec is created but before resources are provisioned so that the benchmark has access to the config and can check conditions before anything is provisioned.

For a short term solution, the check in Prepare is fine. Alternately, you could set ```vms = benchmark_spec.vms[:2]``` to only use the first two VMs.